sros2/sros2/verb/generate_artifacts.py

```diff
@@ -45,7 +47,7 @@ def add_arguments(self, parser, cli_name):
     def main(self, *, args):
         try:
             success = generate_artifacts(
-                args.keystore_root_path, args.node_names, args.policy_files)
+                args.keystore_root_path, args.contexts, args.policy_files)
```

Comment:
```suggestion
                args.keystore_root_path, args.security_contexts, args.policy_files)
```

Reply:
Addressed in https://github.com/ros2/sros2/pull/177/commits/103e18c1366ed793c6269f2d9adab8a16aaf7046.